tests/main/prefer/task.yaml

```diff
@@ -1,5 +1,13 @@
 summary: Simple snap prefer test
 
+details: |
+    Snapd allows selection of preferred source of aliases for snap applications.
+    The "snap prefer" command can override manually created aliases as well as
+    default aliases of other snaps.
+
+    The test installs a snap with automatically enabled aliases, uses the "snap unalias" command
+    to remove them, then
```

Comment:
Unexpected EOF :)

Reply:
Oh, what has happened here. Did I get out of insert mode and nuke some of the content while writing a commit message. Thank you for spotting this.

Reply:
Fixed